ElementX/Sources/Screens/Settings/SettingsScreen/View/SettingsScreen.swift

```diff
@@ -164,8 +164,10 @@ struct SettingsScreen: View {
             VStack {
                 versionText
                     .frame(maxWidth: .infinity)
+                    .textSelection(.enabled)
                 
                 context.viewState.deviceID.map(Text.init)
```

Comment:
I'll be honest... I don't like this.
I would rather have:

```suggestion
                Text(context.viewState.deviceID)
```

Reply:
Ah, but its mapping an optional, so its the single line version of 

```
if let deviceID = context.viewState.deviceID {
    Text(deviceID)
}
```

Reply:
Updated 👍